elections.md

```diff
@@ -65,40 +57,80 @@ representation is achieved. If it is impossible to find sufficient members to
+are announced early October, the steering committee should start looking at
+this process by mid-July, planning to kick things off by mid-August. This
+process is articulated in [kubernetes/community/events/elections/README.md].
 
 ### Vacancies
 
-In the event of a resignation or other loss of a steering committee member, the
-candidate with the next most votes from the previous election will be offered
-the seat.  This process will continue until the seat is filled.  In case this
-fails to fill the seat, a special election for that position will be held as
-soon as possible. The same group of people as described in “eligibility for
-voting” will vote in the special election. A committee member elected in a
-special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder. If a continuity
-member resigns or otherwise is lost from the Committee, that position will not
-be filled.
+In the event of a resignation or other loss of a [bootstrap committee member],
+the position will not be refilled.
+
+In the event of a resignation or other loss of an elected steering committee
+member, the candidate with the next most votes from the previous election will
+be offered the seat.  This process will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will
+be held as soon as possible. Eligible voters from the most recent election
+will vote in the special election (ie: eligibility will not be redetermined
+at the time of the special election). A committee member elected in a special
+election will serve out the remainder of the term for the person they are
+replacing, regardless of the length of that remainder.
 
 ### Limiting Corporate Campaigning
 
 To reduce size of company advantages, candidates may not use their companies
 internal or external brand to campaign.  Their employers cannot solicit votes
-on their behalf or sponsor candidates from partner organizations.  Simply put,
+on their behalf or endorse candidates from partner organizations.  Simply put,
 elections highlight individuals outside of their corporate role and should be
 treated as “brand free” activities.
 
-## Emeritus Term
+## Steering Committee and Election Officer Recusal
 
-Members of the steering committee will graduate to becoming Emeritus members of
-the steering committee.
\ No newline at end of file
+Currently serving steering committee members and the appointed election offers
+pledge to recuse themselves from any form of electioneering, including
+campaigning, nominating, or endorsing. We would prefer that the community
+decide without our heavy influence.
+
+Steering committee members _may_ ask other contributors to consider running,
+and they _may_ vote, so long as this information is kept private.
+
+Steering committee members who intend to run for re-election _may_
+self-nominate but are otherwise expected to adhere to this recusal.
```

Comment:
Here's the outcome of steering committee on recusal vs. re-election: you can self-nominate, but no campaigning

ref: https://groups.google.com/a/kubernetes.io/forum/#!topic/steering/sZPYaEKENzY